test/test_dataloader.py

```diff
@@ -759,13 +759,16 @@ def setUp(self):
         self.persistent_workers = False
 
     def _get_data_loader(self, *args, **kwargs):
-        if (self.persistent_workers or
-                ('persistent_workers' in kwargs and kwargs['persistent_workers'])):
+        if 'persistent_workers' in kwargs:
+            persistent_workers = kwargs['persistent_workers']
+        else:
+            persistent_workers = self.persistent_workers
+        if persistent_workers :
             if (('num_workers' in kwargs and kwargs['num_workers'] == 0)
```

Comment:
:) one minor style recommendation: test `if kwargs.get('num_workers', 0) == 0`.

also just to be safe, can we only allow a single `dataset` as the positional argument, i.e., `def _get_data_loader(self, dataset, **kwargs):`? This guarantees that we can test `kwargs` for things like `num_workers`.

Reply:
Sorry I overlooked this, let me fix it :)